src/main/java/seedu/storemando/logic/parser/SortCommandParser.java

```diff
@@ -18,11 +19,14 @@
      */
     public SortCommand parse(String args) throws ParseException {
         String trimmedArgs = args.trim();
-        if (!trimmedArgs.equalsIgnoreCase(QUANTITY_KEYWORD)
+        if (!trimmedArgs.equalsIgnoreCase(QUANTITY_ASCENDING_KEYWORD)
```

Comment:
Maybe instead change it to 3 positive path conditions in if elseif, throw in else. :)